ansible/roles/sshkey-host-access/README.md

```diff
@@ -0,0 +1,31 @@
+sshkey-host-access
+=========
+
+Create and move keypairs to new instances for access through Web Shells.
+
+Role Variables
+--------------
+
+| Variable                | Required | Default | Choices                   | Comments                                 |
+|-------------------------|----------|---------|---------------------------|------------------------------------------|
+| ATMOUSERNAME            | yes      |         |                           | Username of user getting the keys        |
+| GATEONE_LOCAL_PUBKEY_DIR| no       | `/tmp/`    |                           | the directory to transfer the public key locally |
```

Comment:
I left that in because I am not sure what `GATEONE_LOCAL_PUBKEY_DIR` is for. It is not defined in defaults and I do not see it in variable repos.

After looking into it a little more:
The next task related to this one is named `grab local copy of user ssh public key` and defines a var by finding the key in `/tmp` (which is hard-coded). Since `GATEONE_LOCAL_PUBKEY_DIR` is not defined and defaults to `/tmp/`. I think I should:
1. rename the var like you mention
2. add the variable to the task mentioned above
3. put it in `defaults/main.yml` to avoid confusion in the future